WorkloadManagementSystem/Client/JobReport.py

```diff
@@ -132,20 +137,12 @@ def sendStoredJobParameters(self):
     """ Send the job parameters stored in the internal cache
     """
 
-    parameters = []
-    for pname, value in self.jobParameters.items():
-      pvalue, _timeStamp = value
-      parameters.append([pname, pvalue])
-
+    parameters = [[pname, value[0]] for pname, value in self.jobParameters.items()]
     if parameters:
       result = JobStateUpdateClient().setJobParameters(self.jobID, parameters)
-      if not result['OK']:
-        return result
```

Comment:
Why did you remove these 2 lines above?

Reply:
because there is a `return result` below